sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java

```diff
@@ -150,17 +161,23 @@ public Certificate engineGetCertificate(String alias) {
     public String engineGetCertificateAlias(Certificate cert) {
         String alias = null;
         if (cert != null) {
-            if (aliases == null) {
-                aliases = keyVaultClient.getAliases();
-            }
-            for (String candidateAlias : aliases) {
+            List<String> portalAlias = keyVaultCertificates.getAliases();
+            List<String> aliasList = Stream.of(portalAlias, classpathCertificates.getAliases())
```

Comment:
Can we use `keyVaultCertificates.getAliases()` directly?

Furthermore, Write an interface, named `AzureCertificates`, and both `keyVaultCertificates` and `classpathCertificates` implement this interface. Then we can use stream api in many places.